README.md

```diff
@@ -1,44 +1,51 @@
-Bitcoin Core integration/staging tree
+BitZeny Core integration/staging tree
 =====================================
 
-[![Build Status](https://travis-ci.org/bitcoin/bitcoin.svg?branch=master)](https://travis-ci.org/bitcoin/bitcoin)
+[![Build Status](https://travis-ci.org/BitzenyCoreDevelopers/bitzeny.svg?branch=z2.0.x)](https://travis-ci.org/BitzenyCoreDevelopers/bitzeny)
 
-https://bitcoincore.org
+original site
+----------------
+https://BitZeny.org
```

Comment:
Can't access via HTTPS

Reply:
Thank you for the review.
Fix it and PR again.